src/app/helptext/sharing/afp/afp.ts

```diff
@@ -2,6 +2,19 @@ import { Validators } from "@angular/forms";
 import { T } from "app/translate-marker";
 
 export const helptext_sharing_afp = {
+  fieldset_general: T('General Options'),
+  fieldset_permissions: ('Permissions'),
+  fieldset_allow: T('Allow'),
+  fieldset_deny: T('Deny'),
+  fieldset_ro: T('Read Only'),
+  fieldset_rw: T('Read/Write'),
+  fieldset_hostsallow: T('Hosts Allow'),
+  fieldset_hostsdeny: T('Hosts Deny'),
+  fieldset_other: T('Other Options'),
+
+  placeholder_user_or_group: T('User or Group'),
+  placeholder_host_or_ip: T('Hostname or IP address'),
```

Comment:
We probably should just standardize on "Hostname" everywhere and just point out in the docs that it means either a DNS hostname or an IP address.  But that's going to be big project.